.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: "Release Documentation"
+
+"on":
+  push:
+    branches:
+      - 'develop'
+      - '#5670'
+
+jobs:
+  gen-docs:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Check out vega code
+        uses: actions/checkout@v3
+        with:
+          path: 'vega'
+      -
+        name: "Setup ssh key for PR"
+        uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: ${{ secrets.DOCUMENTATION_KEY }}
+      -
+        name: Check out docs code
+        uses: actions/checkout@v3
+        with:
+          path: 'documentation'
+          repository: 'vegaprotocol/documentation'
+          ssh-key: ${{ secrets.DOCUMENTATION_KEY }}
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      -
+        name: Get dependencies
+        run: cd vega; bash script/gettools.sh
+      -
+        name: Generate docs
+        run: cd vega; make proto_docs; echo "generated docs:"; find protos/generated -type f
+      -
+        name: Make PR to the docs repo
+        env:
+          GH_TOKEN: ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }}
+        run: |
+          mkdir -p documentation/specs/$GITHUB_REF_NAME
+          cp -r vega/protos/generated/* documentation/specs/$GITHUB_REF_NAME/
+          ls -al documentation/specs/$GITHUB_REF_NAME/
+          cd documentation
+          git config --global user.email "vega-ci-bot@vega.xyz"
+          git config --global user.name "vega-ci-bot"
+          branch_name="$(date +%d-%m-%Y--%H-%M)-spec-update"
+          git checkout -b "$branch_name"
+          commit_msg="Automated specs update from vegaprotocol/vega#$GITHUB_SHA"
+          git add specs
+          git commit -m "$commit_msg"  || exit 0
+          git push -u origin "$branch_name"
+          pr_url="$(gh pr create --title '$commit_msg to $GITHUB_REF_NAME' --body 'This is automated pull request that will be merged after pipeline passes')"
```

Comment:
This is not interpreting correctly, https://github.com/vegaprotocol/documentation/pull/216 has the title '$commit_msg to $GITHUB_REF_NAME'

Reply:
Yeah I will fix the quotes